pkg/graph/schema.graphql

```diff
@@ -221,6 +221,22 @@ type UpdateAccessibilityRequestStatusPayload {
   userErrors: [UserError!]
 }
 
+"""
+Parameters for updating a 508/accessibility request's associated CEDAR system
+"""
+input UpdateAccessibilityRequestCedarSystemInput {
+  id: UUID!
+  cedarSystemId: String
```

Comment:
I think in this case we can make `cedarSystemId` required, since without it you're doing a no-op.

Reply:
@ClayBenson94 My thinking was that since cedarSystemId is nullable on a 508 request, you may want to be able to set it to null. I suppose it wouldn't make a lot of sense to do that in practice though.

Reply:
Yeah - there _may_ be a use case for that at some point in the future, but at that point we can probably just make this change then (or make a mutation to explicitly remove the association if we end up needing it)